README.md

```diff
@@ -6,14 +6,24 @@
 `Dockerfilelint` is an node module that analyzes a Dockerfile and looks for common traps, mistakes and helps enforce best practices:
 
 ## Testing
-Start unit tests with `npm test` or `yarn run test`
+Start unit tests with `npm test`, `yarn run test`, or `docker-compose -f docker-compose.test.yml up`
 
 ## Running
 #### From the command line:
 ```shell
 ./bin/dockerfilelint <path/to/Dockerfile>
 ```
 
+#### Command Line options
+```shell
+./bin/dockerfilelint --help
+
+Options:
```

Comment:
Let's not forget to update this based on whatever yargs code we end up with in bin/dockerfilelint.